### ISSUE TITLE
Edge version 17134 not supported

### ISSUE THREAD

**Issue by rogierschouten** [vvo/selenium-standalone#373]
## Expected Behavior
downloads the latest edge driver if specified

## Current Behavior

you get an error saying that the edge version is not valid

## Steps to Reproduce

* install selenium-standalone v6.15.0
* try to start a grid node with edge version 17134

## Your Environment
* Version of `selenium-standalone` that you are using: 6.15.0
* System/platform: Windows 10
* Other details that might be important:

## Solution

The solution is to add the following lines to microsoft-edge-releases.js:

```javascript
,
  '17134': {
    url: 'https://download.microsoft.com/download/F/8/A/F8AF50AB-3C3A-4BC4-8773-DC27B32988DD/MicrosoftWebDriver.exe',
    extension: 'exe'
  }
```

**Comment by serbanghita** [comment]
@rogierschouten 👍 fixed in `6.15.1`

**Comment by imbenwolf** [comment]
please add version 14393 aswell
